src/test/scala/ai/privado/passes/SQLParserTest.scala

```diff
@@ -25,17 +25,21 @@ package ai.privado.passes
 
 import ai.privado.cache.RuleCache
 import ai.privado.model.sql.SQLQueryType
+import ai.privado.semantic.Language.*
 import better.files.File
 import io.joern.jssrc2cpg.Config
 import io.joern.x2cpg.X2Cpg
 import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.semanticcpg.language.*
 import org.scalatest.BeforeAndAfterAll
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
-import ai.privado.semantic.Language._
-import io.shiftleft.semanticcpg.language._
 
+import scala.collection.mutable
 class SQLParserTest extends AnyWordSpec with Matchers with BeforeAndAfterAll {
+  private val cpgs        = mutable.ArrayBuffer.empty[Cpg]
+  private val outPutFiles = mutable.ArrayBuffer.empty[File]
+  private val inputDirs   = mutable.ArrayBuffer.empty[File]
```

Comment:
This can be part of a Base Test case as this will be needed for future test cases as well

Reply:
Moved to base class: ea366f1f6499723e5f9c5835a13da604a66f9a17